__tests__/app.test.js

```diff
@@ -0,0 +1,85 @@
+const app = require("../app");
+const connection = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => connection.end());
+
+describe("GET /api/topics", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((result) => {
+        let { body } = result;
+        expect(Array.isArray(body.topics)).toBe(true);
+      });
+  });
+  it("should return an array of objects with the correct keys", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((result) => {
+        let { body } = result;
+
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("should return an object", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response).toBe("object");
+      });
+  });
+  it("should return an object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(body.article[0]).toEqual(
+          expect.objectContaining({
```

Comment:
This test does not test to check if the correct review has been returned. Consider checking that the title matches the title, and so on for all relevant keys